apis/swagger.yml

```diff
@@ -1585,6 +1585,8 @@ definitions:
       Size:
         description: "size of image's taking disk space."
         type: "integer"
+      Config:
+        $ref: "#/definitions/ContainerConfig"
```

Comment:
I think currently this API struct has more inconsistency with Moby's. We need to make this compatible in a follow-up. @zeppp